pkg/client/unversioned/testclient/testclient.go

```diff
@@ -313,10 +330,6 @@ func (c *FakeExperimental) DaemonSets(namespace string) client.DaemonSetInterfac
 	return &FakeDaemonSets{Fake: c, Namespace: namespace}
 }
 
-func (c *FakeExperimental) HorizontalPodAutoscalers(namespace string) client.HorizontalPodAutoscalerInterface {
```

Comment:
Not changing as explained in https://github.com/kubernetes/kubernetes/pull/20501#issuecomment-180260757